ondewo/csi/conversation.proto

```diff
@@ -116,3 +121,13 @@ message S2sStreamResponse {
         ondewo.t2s.SynthesizeResponse synthetize_response = 2;
     }
 }
+
+message CheckHealthRequest {
+
+}
+
+message CheckHealthResponse {
```

Comment:
Maybe CheckUpstreamHealth or something like that would be less confusing, because you will also have a healthcheck which just does 'list' locally right? Or did you say you will use this even in the dockerfile?

Reply:
I use it in the dockerfile too. I cn of course rename it to `CheckUpstreamHealth` and use just list in the dockerfile, if tht makes sense.